schemas/1.0/general/decision.json

```diff
@@ -1,65 +1,145 @@
 {
-  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
We don't typically update this value, is this required?

Reply:
The JSON generator (`JSON.pretty_generate`) updated the schema here. It's a more recent version. We already have a few schemas using this version.